modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -3,441 +3,523 @@
 
-.. Add a new controller to the project to watch and reconcile the `Memcached` resource:
+.. In the `config/default/manager_auth_proxy_patch.yaml` file, change the `image` value from:
 +
 [source,terminal]
 ----
-$ operator-sdk add controller \
-    --api-version=cache.example.com/v1alpha1 \
-    --kind=Memcached
-----
-+
-This scaffolds a new controller implementation under `pkg/controller/memcached/`.
-
-.. For this example, replace the generated controller file `pkg/controller/memcached/memcached_controller.go` with the link:https://github.com/operator-framework/operator-sdk/blob/master/example/memcached-operator/memcached_controller.go.tmpl[example implementation].
-+
-The example controller executes the following reconciliation logic for each `Memcached` resource:
-+
---
-* Create a Memcached deployment if it does not exist.
-* Ensure that the Deployment size is the same as specified by the `Memcached` CR spec.
-* Update the `Memcached` resource status with the names of the Memcached pods.
---
-+
-The next two sub-steps inspect how the controller watches resources and how the reconcile loop is triggered. You can skip these steps to go directly to building and running the Operator.
-
-.. Inspect the controller implementation at the `pkg/controller/memcached/memcached_controller.go` file to see how the controller watches resources.
-+
-The first watch is for the `Memcached` type as the primary resource. For each add, update, or delete event, the reconcile loop is sent a reconcile `Request` (a `<namespace>:<name>` key) for that `Memcached` object:
-+
-[source,go]
-----
-err := c.Watch(
-  &source.Kind{Type: &cachev1alpha1.Memcached{}}, &handler.EnqueueRequestForObject{})
+gcr.io/kubebuilder/kube-rbac-proxy:<tag>
 ----
 +
-The next watch is for `Deployment` objects, but the event handler maps each event to a reconcile `Request` for the owner of the deployment. In this case, this is the `Memcached` object for which the deployment was created. This allows the controller to watch deployments as a secondary resource:
+to use the supported image:
 +
-[source,go]
+[source,terminal,subs="attributes+"]
 ----
-err := c.Watch(&source.Kind{Type: &appsv1.Deployment{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &cachev1alpha1.Memcached{},
-	})
+registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.6
 ----
 
-.. Every controller has a `Reconciler` object with a `Reconcile()` method that implements the reconcile loop. The reconcile loop is passed the `Request` argument which is a `<namespace>:<name>` key used to lookup the primary resource object, `Memcached`, from the cache:
+. Update the `test` target in your Makefile to install dependencies required during later builds by replacing the following lines:
 +
-[source,go]
+.Existing `test` target
```

Comment:
The `collapsible` setting a bit wonky in these examples, it's just boxing the code blocks.